src/osx/carbon/utilscocoa.mm

```diff
@@ -203,10 +203,23 @@ wxBitmap wxOSXCreateSystemBitmap(const wxString& name, const wxString &client, c
 
 #if wxOSX_USE_COCOA
 
-wxBitmap wxOSXCreateSystemBitmap(const wxString& name, const wxString &WXUNUSED(client), const wxSize& WXUNUSED(size))
+WX_NSImage wxOSXGetSystemImage(const wxString& name)
 {
     wxCFStringRef cfname(name);
-    return wxBitmap( [NSImage imageNamed:cfname.AsNSString()] );
+    NSImage* nsimage = [NSImage imageNamed:cfname.AsNSString()];
+    return nsimage;
+}
+
+wxBitmap wxOSXCreateSystemBitmap(const wxString& name, const wxString &client, const wxSize& sizeHint)
+{
+    NSImage* nsimage = wxOSXGetSystemImage(name);
+    if ( nsimage )
+    {
+        // if ( sizeHint != wxDefaultSize )
```

Comment:
It would be be better to avoid checking in commented out code.

Reply:
Yes, I'll do so, I'm not sure yet on which level a resize (if at all) should occur

Reply:
FWIW, I think that if it's possible not to do it all, it would be preferable.